apps/frontend/src/renderer/components/github-prs/components/PRLogs.tsx

```
@@ -235,20 +316,205 @@ function PhaseLogSection({ phase, phaseLog, isExpanded, onToggle, isStreaming =
+          >
+            {displayName}
+          </Badge>
+          {hasMultipleEntries && (
+            <button
+              onClick={onToggle}
+              className={cn(
+                'flex items-center gap-1 text-[10px] px-2 py-0.5 rounded transition-colors',
+                'text-muted-foreground hover:text-foreground hover:bg-secondary/50',
+                isExpanded && 'bg-secondary/50 text-foreground'
+              )}
+            >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)